src/components/Datepicker/Views/Days.tsx

```diff
@@ -0,0 +1,75 @@
+import type { FC } from 'react';
+import { twMerge } from 'tailwind-merge';
+import { mergeDeep } from '~/src/helpers/merge-deep';
+import { useTheme, type DeepPartial } from '../..';
+import { useDatePickerContext } from '../DatepickerContext';
+import { addDays, getFirstDayOfTheMonth, getFormattedDate, getWeekDays, isDateEqual, isDateInRange } from '../helpers';
+
+export interface FlowbiteDatepickerViewsDaysTheme {
+  header: {
+    base: string;
+    title: string;
+  };
+  items: {
+    base: string;
+    item: {
+      base: string;
+      selected: string;
+      disabled: string;
+    };
+  };
+}
+
+export interface DatepickerViewsDaysProps {
+  theme?: DeepPartial<FlowbiteDatepickerViewsDaysTheme>;
+}
+
+export const DatepickerViewsDays: FC<DatepickerViewsDaysProps> = ({ theme: customTheme = {} }) => {
+  const theme = mergeDeep(useTheme().theme.datepicker.views.days, customTheme);
+
+  const { weekStart, minDate, maxDate, viewDate, selectedDate, changeSelectedDate, language } = useDatePickerContext();
+
+  const weekDays = getWeekDays(language, weekStart);
+  const startDate = getFirstDayOfTheMonth(viewDate, weekStart);
+
+  return (
+    <>
+      <div className={theme.header.base}>
+        {weekDays.map((day, index) => (
+          <span key={index} className={theme.header.title}>
+            {day}
+          </span>
+        ))}
+      </div>
+      <div className={theme.items.base}>
+        {[...Array(42)].map((_date, index) => {
+          const currentDate = addDays(startDate, index);
+          const day = getFormattedDate(language, currentDate, { day: 'numeric' });
+
+          const isSelected = isDateEqual(selectedDate, currentDate);
+          const isDisabled = !isDateInRange(currentDate, minDate, maxDate);
+
+          return (
+            <button
```

Comment:
Missing aria atributes for screen readers:

- aria-label (like "Choose August 12, 2023")
- aria-selected
- aria-disabled
- role="option"